docker/ci-docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  lib-publisher:
+    build:
+      dockerfile: docker/Dockerfile
+      context: ../
+      args:
+        MAVEN_USER: ${MAVEN_USER}
+        MAVEN_PASSWORD: ${MAVEN_PASSWORD}
+    image: copperleaf/spring-social-github:${GIT_COMMIT:-dev}
```

Comment:
Same here -- can we somehow configure the version using `gradle.properties` and not from the Git commit (or `-dev` which I don't believe would be considered a valid Java package version)?